collector/__init__.py

```diff
@@ -337,28 +345,34 @@ def _calculate_target_info(self,
         rem_visibility_frac_numerator = obs.exec_time() - obs.total_used()
 
         for ridx, jday in enumerate(reversed(self.time_grid)):
```

Comment:
Are we calculating visibility for the whole timegrid it seems, does it takes too much time? I wonder as Bryan used to parallelize that to gain speed and maybe is something we might want to add in the future.

Reply:
Yes, we plan to parallelize as we are calculating visibility across all base targets for all observations across the entire time grid, so it'll be essential when we work with a larger data set. I'll add a Jira task for that so we don't forget to add that.